### ISSUE TITLE
Github workflow routes_location doesn't work as expected

### ISSUE THREAD

**Issue by itpropro** [Azure/static-web-apps#372]
The property `routes_location` does not seem to have an impact on the new `staticwebapp.config.json`. From my tests, this file is exclusively looked for (by the Github Action/StaticSitesClient) in the folder that is specified in the appLocation and it's subfolders. 
Currently the `routes_location` property is only considered when using the old `routes.json` config file, the new `staticwebapp.config.json` only works when located in the app folder.
Can you confirm this @mkarmark and would it be possible to provide a seconds property like `swaconfig_location` or one property that is used in both cases?

**Comment by itpropro** [comment]
Hey,
It's just confusing that the documentation states otherwise. It would be nice to have the possibility to use an alternate location like the repository root, to keep the configuration and SWA related tooling separate from the actual frontend.

**Comment by anthonychu** [comment]
> It's just confusing that the documentation states otherwise. 

@itpropro Thanks for pointing [this](https://docs.microsoft.com/en-us/azure/static-web-apps/github-actions-workflow#route-file-location) out. We'll get it updated.

**Comment by joelcoxokc** [comment]
@miwebst Yes there is a scenario where I would like to change this. 

Does the routes_location actually work?

I am using a mono-repo, and for some reason the github action is only finding one routes file for all my apps, instead of using the `staticwebapp.config.json` for each app. 

will `routes_location` solve this?

**Comment by itpropro** [comment]
Currently it will not solve this. The ‘routes_location’ only works for the old routes config file. The GitHub Action statically looks in the app_location for your staticwebapp.config.json.

**Comment by joelcoxokc** [comment]
@miwebst 
 Is this the other issue you were referring to?
#416 


Can we add the `config_location` property to support angular mono-repos?

**Comment by miwebst** [comment]
Yes this is the one, we are working on a fix. The initial fix will be to make routes_location work with staticwebapp.config.json. Long term we may add config_location if needed.

**Comment by joelcoxokc** [comment]
@miwebst Wonderful. You all are awesome!

**Comment by miwebst** [comment]
Ok, if you are using GitHub, you can now use this: config_file_location to point to your SWA config. We will be getting this into DevOps too.

**Comment by caitlinwillmot** [comment]
> 
> 
> Ok, if you are using GitHub, you can now use this: config_file_location to point to your SWA config. We will be getting this into DevOps too.

Hi, is there an ETA on when this will be available in DevOps? Thanks.
